posts/lte-examples.md

```diff
@@ -0,0 +1,690 @@
+(ulift.{1}  
+  { f : S → X | ∃ (c : ℝ≥0) (g : S → filtration X c), 
+    continuous g ∧ f = coe ∘ g }) := 
+rfl
+```
+If `S : Profinite.{0}` and `X : Condensed.{0} Ab.{1}`, then the notation `Γ_ S X` appearing in the second line should be read as $\Gamma(S,X)$, i.e. the sections of `X` over `S`.
+
+Since Lean's type theory does not have cumulative universes, the definition of `CompHausFiltPseuNormGrp.to_Condensed` involves a universe bump using `ulift`, in order to obtain an object of `Ab.{1}` as opposed to `Ab.{0}` (see the discussion above).
+Putting that aside, the sections $\Gamma(S,X)$ of the condensed abelian group associated to a CHFPNG $X$ over a profinite set $S$ is the set of functions $f : S \to M$ which factor as the composition of a continuous map $g : S \to M_c$ and the natural inclusion $\mathtt{coe} : M_c \to M$ for some $c$.
+The group structure on this set of sections is the obivous one, given by pointwise addition.
+```lean
+example (X : CompHausFiltPseuNormGrp.{0}) (S : Profinite.{0})
+  (f g : Γ_ S (CompHausFiltPseuNormGrp.to_Condensed X)) (s : S) :
+  (f + g) s = f s + g s := rfl
+```
+
+## $p$-Radon measures
+Fix a real number $p$ satisfying $0 < p \le 1$.
+We will now discuss the relationship between $\mathcal{M}\_{p}(S)$ and the space of $p$-Radon measures.
+First we address the question of actually defining the space of $p$-Radon measures.
+
+Given any `S : Profinite.{0}`, and $p$ as above, we define `S.Radon_png p`, an object of `CompHausFiltPseuNormGrp₁`, as follows.
+As a set, it consists of all continuous linear maps $\mu : C(S,\mathbb{R}) \to \mathbb{R}$ such that there exists some $C \in \mathbb{R}_{\geq 0}$ satisfying the following condition: For any partition $S = V_1 \cup \cdots \cup V_n$ into disjoint clopen sets, one has 
+$$ \sum_i |\mu(I_i)|^p \le C, $$
+where $I_i \in C(S,\mathbb{R})$ denotes the (continuous) indicator function of $V_i$.
+Since $S$ is compact, Hausdorff and totally disconnected, one may verify that this agrees with the usual space of signed $p$-Radon measures (which reduces to the usual notion of a [signed Radon measure](https://en.wikipedia.org/wiki/Radon_measure) when $p = 1$); see also Exercise 3.3 of [`Analytic.pdf`](https://www.math.uni-bonn.de/people/scholze/Analytic.pdf). 
+
+The $c$-th part of the filtration on `S.Radon_png p` is given by those $\mu$ where the sums above are all bounded by `c`. 
+If one endowes the continuous dual with the weak topology, this subspace agrees with the corresponding subspace of the space of signed $p$-Radon measures, and is a compact Hausdorff space.
+From here, it is straightforward to see that `S.Radon_png p` is a CHFPNG.
+
+We added several examples in the file `examples/radon_measures.lean` dedicated to this object `S.Radon_png p`.
+First, any element of `S.Radon_png p` can be considered as a continuous functional on $C(S,\mathbb{R})$.
+```lean
+example (S : Profinite.{0}) (μ : S.Radon_png p) : C(S,ℝ) →L[ℝ] ℝ := μ.1
```

Comment:
Mention that `Radon_png` is also defined in terms of dependent pairs, similarly to `C(X,Y)` and/or Sheaves.